Cargo.toml

```diff
@@ -32,8 +32,9 @@ license-file = ["LICENSE"]
 depends = "$auto, systemd"
 section = "utils"
 priority = "optional"
+conf-files = ["/etc/uption/uption.toml", "/etc/systemd/system/uption.service"]
 assets = [
-    ["target/release/uption", "usr/bin/", "755"],
```

Comment:
How did this work before? 😅

Reply:
I don't know :laughing: Maybe to current folder is `/` when the debian package is created.